src/pages/workspace/companyCards/WorkspaceCompanyCardsPage.tsx

```diff
@@ -119,6 +121,7 @@ function WorkspaceCompanyCardsPage({route}: WorkspaceCompanyCardsPageProps) {
                             policyID={policyID}
                             onAssignCard={assignCard}
                             isAssigningCardDisabled={isAssigningCardDisabled}
+                            isLoadingCardsList={isLoadingCardsList}
```

Comment:
Rather than passing this as a prop, can you please copy/move everything necessary inside of `WorkspaceCompanyCardsList`?